createdisk-library.sh

```diff
@@ -210,11 +210,15 @@ function prepare_qemu_guest_agent() {
     local vm_ip=$1
 
     # f36 default selinux policy blocks usage of qemu-guest-agent over vsock
-    ${SCP} qemuga-vsock.te core@${vm_ip}:
-    ${SSH} core@${vm_ip} '/usr/bin/checkmodule -M -m -o qemuga-vsock.mod qemuga-vsock.te'
-    ${SSH} core@${vm_ip} '/usr/bin/semodule_package -o qemuga-vsock.pp -m qemuga-vsock.mod'
+    if ! checkmodule -c 19 2>&1 |grep 'invalid option' >/dev/null; then
+	    # RHEL8 checkmodule does not have this arg
+	    MOD_VERSION_ARG="-c 19"
+    fi
```

Comment:
This block not work as expected because as per man page `The  return status of a pipeline is the exit status of the last command, unless the pipefail option is enabled.  If pipefail is enabled, the pipeline's return status is the value
       of the last (rightmost) command to exit with a non-zero status, or zero if all commands exit successfully. `  in our case pipefail option is set for the script.

```
$ cat ./test.sh 
#!/bin/bash

set -eou pipefail

if ! checkmodule -c 19 2>&1 | grep 'invalid option' >/dev/null; then
	    # RHEL8 chAeckmodule does not have this arg
	    MOD_VERSION_ARG="-c 19"
	    echo "added -c 19"
fi

echo ${MOD_VERSION_ARG}

$ ./test.sh 
added -c 19
-c 19
```

Reply:
Not sure what the best way around this is?
Wrap it in `set +o pipefail`?

```
    set +o pipefail
    if ! checkmodule -c 19 2>&1 |grep 'invalid option' >/dev/null; then
	    # RHEL8 checkmodule does not have this arg
	    MOD_VERSION_ARG="-c 19"
    fi
    set -o pipefail
```

Reply:
@cfergeau yes I think we should do that and add comment why we are unsetting pipefail for this block of code.